spec/factories/occupation_standards.rb

```diff
@@ -3,5 +3,6 @@
     occupation
     url { "http://example.com" }
     registration_agency
+    organization
```

Comment:
Since an organization is not always required for an occupation standard, let's leave this off the factory. In general factories should just contain the minimal fields required. It will be helpful to keep our test suite running fast by not having it create extra organization records every time it creates an occupation standard record.